### ISSUE TITLE
Missing explore in new loomio?

### ISSUE THREAD

**Issue by peteruithoven** [loomio/loomio#2526]
Besides going directly to loomio.org/explore I can't figure out how to explore / find public groups in the new Loomio interface. 
In the old interface I get there through:
- *Groups* button > *Find a group*
- In the footer, the *Public groups* button
Is there an alternative I'm missing? Is this something that is still being figured out?

**Comment by rdbartlett** [comment]
That's because we are planning to make the Explore page much much better
before we point a lot more people at it.

Do you mind saying what you like about the Explore page currently?

**Comment by peteruithoven** [comment]
That clarifies things. Nothing in particular, but I was trying to find a existing group I didn't join yet and thus I got kind of lost. 
On the switch page you actually do mention "Some less frequently used features may not be there yet.".

To be perfectly honest, I backed the 1.0 project, more than a year ago and I expected this to be there by now, but I also understand how developments go, keep up the good work!
